### PR TITLE
TH2-2370: Tree table card cannot show more than 62 groups

### PR DESCRIPTION
**Pull request by VekesserVladimir**

### CODE REVIEW

File: src/styles/tables.scss [hunk 1]
```diff
@@ -439,7 +365,10 @@
 
 .params-table {
 	@include default-table;
-	@include scrollbar;
+
+	&-wrapper {
+		overflow-x: scroll;
```
Comment (rzvdv) [inline src/styles/tables.scss:370]: isnt auto a better option?

File: src/styles/common/_mixins.scss [hunk 2]
```diff
@@ -203,7 +202,6 @@
 		background-color: $tableBodyBackground;
 		border-right: 0;
 		border-left: 0;
```
Comment (rzvdv) [inline src/styles/common/_mixins.scss:204]: i think this breaks aligning in first column of verification table

**Review by rzvdv — CHANGES_REQUESTED** [review]

File: src/styles/common/_mixins.scss (hunk 2 above)
Comment (VekesserVladimir) [reply src/styles/common/_mixins.scss]: Fixed

File: src/styles/tables.scss (hunk 1 above)
Comment (VekesserVladimir) [reply src/styles/tables.scss]: Fixed

**Review by rzvdv — APPROVED** [review]

**Pull request merged**